odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -866,11 +895,20 @@ PROCEDURE READCONFIGFILESETTINGS
    BNODEBUGMSG="  PID: "+FORMAT("99/99/9999",PARAMINSTALLDATE) CALL BNODEBUGMSGADD
    BNODEBUGMSG="  BNODEBUGMODE: "+BOOLEANCHR(BNODEBUGMODE) CALL BNODEBUGMSGADD
 
+   IF DEFAULTOTAVALUE>=$0.00 THEN
+    DO
+     TMPCHR=FORMAT("###,###,##9.99",DEFAULTOTAVALUE)
+     CALL NLTS
+     BNODEBUGMSG="OTA DEFAULT 'XXXX' AMOUNT $"+TMPCHR
+     CALL BNODEBUGMSGADD
+    END
    FOR TMPLOOP=1 TO PARAMOTACOUNT
     DO
-     BNODEBUGMSG=FORMAT("OTA 999 -",TMPLOOP)+
-                 FORMAT(" SHARE TYPE 9999  ",PARAMOTASHARETYPE(TMPLOOP))+
-                 FORMAT("OT AMOUNT $###,###,##9.99",PARAMOTAAMOUNT(TMPLOOP))
+     TMPCHR=FORMAT("###,###,##9.99",PARAMOTAAMOUNT(TMPLOOP))
+     CALL NLTS
+     BNODEBUGMSG=FORMAT("  OTA 999 -",TMPLOOP)+
```

Comment:
I suggest changing "OTA 999 -" in the configs debug to "OTA 99 -" as 99 is the max

Reply:
@jordan-keenan Done